test/fixtures/o-test/main.js

```diff
@@ -1,5 +1,5 @@
 /*eslint-disable */
 const Test = require('./src/js/test');
 const jsonTest = require('./src/js/json-test.json');
-const textTest = requireText('./src/js/require-text.txt');
+// const textTest = requireText('./src/js/require-text.txt');
```

Comment:
Should this line be deleted instead of commented out?

Reply:
oops https://github.com/Financial-Times/obt5fork/pull/61#issuecomment-663076303